src/styles/main.scss

```diff
@@ -0,0 +1,67 @@
+$day-size: 100px;
+$day-gap: 1px;
+$calendar-border-color: #000;
+$day-default-color: #eee;
+$day-hover-color: #ffbfcb;
+$transition-duration: 0.5s;
+
+body {
+  font-family: Arial, sans-serif;
+  font-size: 300px;
```

Comment:
The font size for the body is set to 300px, which is unusually large and likely an oversight. Consider reviewing this value to ensure it aligns with your design requirements.